src/main/groovy/io/cify/framework/core/Capabilities.groovy

```diff
@@ -55,37 +74,78 @@ class Capabilities {
      */
     DesiredCapabilities toDesiredCapabilities(DeviceCategory category) {
```

Comment:
Is it worth to check that category is from DeviceCategory ?

Reply:
What else can it be? NullPointerException is being caught...